### DISCUSSION TITLE
How to install Payload CMS with NextJS 14 ?

### DISCUSSION THREAD

**Discussion by thibault-chausson** [payloadcms/payload#9440] [General]
I have an application running on NextJS 14, and I would like to install Payload on this application.
How can I do this since the release of Payload V3?

Thanks !!!

**Comment by andershermansen** [comment]
You cannot. You either need to upgrade your application to NextJS 15, or you need to run your application and payload separatly and connect with API. Payload V3 requires NextJS 15.

**Reply by thibault-chausson** [reply]
Can I use Payload V2? 
If so, how do I go about it? The `npx create-payload-app` doesn't offer version 2.

**Reply by andershermansen** [reply]
No, sorry. Payload v2 is an express application, so can also not run on NextJS 14.

I'm unsure how to create payload app with v2, others need to chime in.

**Reply by thibault-chausson** [reply]
Thank you for your reply

**Reply by HarleySalas** [reply]
> Can I use Payload V2? If so, how do I go about it? The `npx create-payload-app` doesn't offer version 2.

I really would not be discouraged by this.

If it's genuinely important for you, it is possible to run v2 and an older version of NextJS as a single process. However, in most V2 setups, you'd be running Payload as a separate process anyways, then running your NextJS application separately. So, you very well could just use Payload V3 as a separate process and build your application in NextJS 14, just like people used to do with V2, before it could run so easily as a single NextJS process.

Particularly with #9463 there will be an SDK for the REST API, which will have full type support, just like the Local API.
